CRM/ACL/API.php

```diff
@@ -211,10 +209,9 @@ public static function groupPermission(
       return NULL;
     }
     if (!$contactID) {
-      $session = CRM_Core_Session::singleton();
       $contactID = NULL;
-      if ($session->get('userID')) {
-        $contactID = $session->get('userID');
+      if (CRM_Core_Session::singleton()->getLoggedInContactID()) {
+        $contactID = CRM_Core_Session::singleton()->getLoggedInContactID();
```

Comment:
Yeah probably - I was trying to keep my replacements very consistent to avoid risk of error on the basis that sort of thing could happen later